data/docs/useAsyncEffect.md

```diff
@@ -12,6 +12,23 @@ This is a version of `useEffect` that accepts an async function.
 
 [//]: # "Main"
 
-## Examples
+```jsx
+import { useDidMount } from "rooks";
+export default function App() {
+  const [value, setValue] = useState(false);
+  useAsyncEffect(async () => {
+    setTimeout(() => {
```

Comment:
Since this is an `async` effect, we want to use a code snippet which can `await` .

Reply:
```jsx
import { useAsyncEffect  } from "rooks";

const foo = new Promise((resolve, reject) => {
  setTimeout(() => {
    resolve("bar");
  }, 300);
});

export default function App() {
  const [value, setValue] = useState(false);
  useAsyncEffect(async () => {
    await foo() 
  }, []);

  return value;
}
```

something like this?